transcriptomics_data_service/config.py

```diff
@@ -1,8 +1,10 @@
-from bento_lib.config.pydantic import BentoBaseConfig
+from bento_lib.config.pydantic import BentoFastAPIBaseConfig
 from fastapi import Depends
 from functools import lru_cache
 from typing import Annotated
 
+from pydantic_settings import BaseSettings, SettingsConfigDict
```

Comment:
is BaseSettings used here?

Reply:
Oops no. I might try out [ruff](https://github.com/astral-sh/ruff), since Black doesn't look for unused imports.
It seems to work well with Black and is used by FastAPI.

Reply:
Adding it to the project with Black compatibility was pretty hassle free!
We can use TDS as the `ruff` guinea pig to see if we like it.

Reply:
I think we don't need both ruff and black, right?

Reply:
Indeed, we could use ruff alone for linting and formating, which would be best I think.
This was mostly a test to check how it worked. I will add ruff formating in the next commit.

Reply:
Now exclusively using Ruff for formatting and linting.